pierone/cli.py

```diff
@@ -300,6 +300,90 @@ def scm_source(config, team, artifact, tag, url, output):
                     max_column_widths={'revision': 10})
 
 
+def query_yes_no(question, default="yes"):
```

Comment:
this function IMHO does not make sense, you can use one of the provided `click` functions: http://click.pocoo.org/5/prompts/